<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QScrollArea" name="radiosBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>150</height>
       </size>
      </property>
      <property name="lineWidth">
       <number>3</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="widgetResizable">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>818</width>
         <height>352</height>
        </rect>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>-20</y>
          <width>821</width>
          <height>371</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="2" column="5" colspan="2">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Control</string>
      </property>
      <widget class="QPushButton" name="loadLogsButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>80</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Load Logs</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>84</width>
         <height>15</height>
        </rect>
       </property>
       <property name="text">
        <string>Coordenadas:</string>
       </property>
      </widget>
      <widget class="QLabel" name="coordsLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>[x,y,z]</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" colspan="6">
     <widget class="QGraphicsView" name="drawBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="midLineWidth">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Zoom</string>
      </property>
      <widget class="QPushButton" name="invertAxisButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>89</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Invert   Y</string>
       </property>
      </widget>
      <widget class="QPushButton" name="increaseZoomButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>50</y>
         <width>30</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QLabel" name="zoomLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>89</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>----------</string>
       </property>
      </widget>
      <widget class="QPushButton" name="decreaseZoomButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>30</width>
         <height>27</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1158</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
